src/odemis/acq/move.py

```diff
@@ -576,6 +586,7 @@ def _doCryoSwitchSamplePosition(future, target):
 
     try:
         if role == "enzel":
+            # TODO enzel code broken as rm is used in other functions insteasd of rz
```

Comment:
Well, let's start by not modifying the enzel code! All the lines below which are in this block should be reverted.